platform/kube/inject/inject.go

```diff
@@ -37,8 +37,8 @@ import (
 // Defaults values for injecting istio proxy into kubernetes
 // resources.
 const (
-	DefaultInitImage            = "docker.io/istio/init:latest"
-	DefaultRuntimeImage         = "docker.io/istio/runtime:latest"
+	DefaultHub                  = "docker.io"
+	DefaultTag                  = "2017-03-22-17.30.06"
```

Comment:
That would work, but maybe we should wait until we have full acceptance test in place which verifies latest version of manager, proxy, and mixer together with a matching version of istioctl?